### PR TITLE
Adds support for multiple profiles in AwsS3CpActivity

### PR DESCRIPTION
**Pull request by pkhetrapal**

### CODE REVIEW

File: contrib/activity/definition/src/main/scala/com/krux/hyperion/activity/AwsS3CpActivity.scala [hunk 1]
```diff
@@ -23,21 +24,33 @@ case class AwsS3CpActivity private(
   def updateShellCommandActivityFields(fields: ShellCommandActivityFields) = copy(shellCommandActivityFields = fields)
 
   def withOverwrite() = copy(isOverwrite = HBoolean.True)
-  def withRecursive() = withAdditionalArguments("--recursive")
-  def withProfile(profile: HString) = withAdditionalArguments("--profile", profile)
```
Comment (realstraw) [inline contrib/activity/definition/src/main/scala/com/krux/hyperion/activity/AwsS3CpActivity.scala]: Please make sure it's backward compatible

File: contrib/activity/definition/src/main/scala/com/krux/hyperion/activity/AwsS3CpActivity.scala [hunk 2]
```diff
@@ -23,21 +24,33 @@ case class AwsS3CpActivity private(
...
+  def withDestinationAdditionalArguments(arguments: HString*) = copy(
+    destinationAdditionalArguments = this.destinationAdditionalArguments ++ arguments
+  )
 
   private val removeScript = if (isOverwrite) s"aws s3 rm --recursive $destinationS3Path;" else ""
 
-  private val s3CpScript = s"aws s3 cp ${additionalArguments.mkString(" ")} $sourceS3Path $destinationS3Path;"
+  private val s3CpScript =
+    s"""
+       |aws s3 cp ${sourceAdditionalArguments.mkString(" ")} $sourceS3Path
```
Comment (realstraw) [inline contrib/activity/definition/src/main/scala/com/krux/hyperion/activity/AwsS3CpActivity.scala]: @arijitdey @pkhetrapal  Are you sure aws command accept two profile arguments? I just double checked, I don't see where it's not documented.

Comment (arijitdey-sfdc) [reply contrib/activity/definition/src/main/scala/com/krux/hyperion/activity/AwsS3CpActivity.scala]: I did test it out with one profile destination account. 

@pketrapal Can you please work with PE to test this use case out? Ask them that you need access to two profiles.

Comment (pkhetrapal) [reply contrib/activity/definition/src/main/scala/com/krux/hyperion/activity/AwsS3CpActivity.scala]: Confirmed with @paul-krohn - it works as expected

**Comment by pkhetrapal** [comment]
Closing this as it is not required anymore

**Pull request closed without merging**